_data/other.yml

```diff
@@ -279,6 +279,16 @@ websites:
       img: promoter.png
       tfa: No
       twitter: Promoter_io
+      
+    - name: Repairshopr
+      url: http://www.repairshopr.com/
```

Comment:
Ahhh - I figured it out.  When you go to https://www.repairshopr.com it works; but I was just going to https://repairshopr.com; it redirects to http://repairshopr.com.  Poor implementation on their part; but I'll update it as soon as I get on a real computer (and not my phone).  :)